### DISCUSSION TITLE
File picker returns nothing when tried to pick files bigger than 50MB

### DISCUSSION THREAD

**Discussion by drclahiru** [capawesome-team/capacitor-plugins#52] [General]
Version:  5.1.0

Platform: Android 13

 `const attachmentFiles: PickFilesResult = await FilePicker.pickFiles({
      multiple: true,
      readData: true,
 })`

File picker returns nothing when tried to pick files bigger than 50MB. Files which are small in size (<10MB) work as it should. But if selected together with a bigger file, again returns nothing. No console error or anything. 

Is this the expected behavior and is there any max file size requirement? Or is this a bug?

**Comment by robingenz** [comment]
If you use `readData: true`, then the files are converted to Base64 strings. With small files this is no problem, but with large files it is (OOM exception). From which file size exactly, that depends on the device, the configuration etc.. I recommend not to use `readData` and to handle everything on native layer.
